Azure/sonic-buildimage
# [sonic-yang-models]: Fix vlan name pattern incorrect issue

* Original pattern can't accept Vlan1000~Vlan3999. Fix it.

**- Why I did it**

Try to fix dynamic breakout port configuration function when there is Vlan1000~Vlan3999 exist.

**- How I did it**

Modify the yang model pattern of the vlan name string

**- How to verify it**

1. Create a vlan with name, Vlan1000
```config vlan add 1000```

2. Breakout the port
```config interface breakout Ethernet0 4x25G```

3. It should able to breakout the port

**- Which release branch to backport (provide reason below if selected)**

**- A picture of a cute animal (not mandatory but encouraged)**
